test/mosquito/metadata_test.cr

```diff
@@ -0,0 +1,60 @@
+require "../test_helper"
+
+describe Mosquito::Metadata do
+  getter(store_name : String) { "test_store#{rand 1000}" }
+  getter(store : Metadata) { Metadata.new store_name }
+  getter(field : String) { "foo#{rand 1000}" }
```

Comment:
I had no idea you could do this. I hate it... but I also didn't know about it 😉

Reply:
well... if I find a solution I don't hate I'll let you know :D

Reply:
I kinda like it actually 😅  I might steal this for some of my code.